app/src/main/java/com/nervousfish/nervousfish/exceptions/DatabaseException.java

```diff
@@ -0,0 +1,13 @@
+package com.nervousfish.nervousfish.exceptions;
+
+/**
+ * Thrown when there is a issue with the database
+ */
+public class DatabaseException extends RuntimeException {
+    /**
+     * {@inheritDoc}
+     */
+    public DatabaseException(final String msg) {
```

Comment:
Should `@override` when using `{@inheritDoc}`, right (applies to DeserializationException.java as well)?